main.cpp

```diff
@@ -165,6 +166,10 @@ int main(void)
         return -1;
     }
 
+    // Initialize root of trust
+    DeviceKey &devkey = DeviceKey::get_instance();
+    devkey.generate_root_of_trust();
```

Comment:
I think you can do
```cpp
#if MBED_MAJOR_VERSION >= 6
    devkey.generate_root_of_trust();
#else
    // Try generating dummy key just to force DeviceKey to create a random key
    // in case it does not exist yet
    char dummy[16];
    DeviceKey::generate_derived_key("SALT", 4, dummy, DEVICE_KEY_16BYTE);
#endif
```

And it should work against 5.x, 6.x and master.

Reply:
Would it actually be even simpler to just have the version check:
```
#if MBED_MAJOR_VERSION >= 6
    devkey.generate_root_of_trust();
#endif
```
So this would work just like it did before in Mbed OS 5, and the new feature would be only enabled in new Mbed OS version.

Reply:
That is OK as well.. I just thought that you want to somehow force the key generation.